lldb/test/API/python_api/type/main.cpp

```diff
@@ -8,6 +8,9 @@ class Task {
         TASK_TYPE_1,
         TASK_TYPE_2
     } type;
+    enum E : unsigned char {} e;
+    union U {
+    } u;
```

Comment:
Can you move these below that `struct name`. Just so it's closer to Jim's example where if we did a depth-first search we would've found `name::E`

Reply:
Done!